src/Microsoft.Extensions.Caching.SqlConfig.Tools/Microsoft.Extensions.Caching.SqlConfig.Tools.csproj

```diff
@@ -19,7 +19,13 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.CommandLineUtils.Sources" Version="$(AspNetCoreVersion)" PrivateAssets="All" />
-    <PackageReference Include="System.Data.SqlClient" Version="$(CoreFxVersion)" />
+
+    <!--
+      * Use 4.4.0-* instead of $(CoreFxVersion) to address "SqlClient fails with netcoreapp2.0 on Win7/Server2008"
```

Comment:
This change should be temporary if we eventually update `$(CoreFxVersion)` to `4.4.0-*`, though this may or may not be happening for 2.0.  This change has already been merged like this in several other repos, and I'd like to keep this as simple as possible, so I think this is good enough for preview1.  I will create an issue to convert these to `dependencies.props` in dev.